WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md

```diff
@@ -0,0 +1,19 @@
+SSL/TLS certificates are crucial for securing network communications. They authenticate the identity of servers and enable encrypted connections between clients and servers. However, improperly configured or invalid certificates can lead to severe security vulnerabilities.
+
+Here are key aspects of SSL certificate validation:
+
+1. **Expiration Management:** An expired certificate can no longer be trusted and may lead to security warnings or connection failures.
+
+2. **Renewal Alerts:** Certificates approaching their expiration date (within 30 days) should be renewed promptly to avoid service interruptions.
+
+3. **Key Strength:** Certificates that use RSA keys shorter than 2048 bits are considered weak and vulnerable to brute-force attacks.
+
+4. **Signature Algorithm Integrity:** Outdated algorithms like MD5 or SHA1 are vulnerable to collision attacks and should not be used.
+
+5. **Hostname Verification:** The certificate's Common Name (CN) or Subject Alternative Name (SAN) should match the target hostname to prevent potential man-in-the-middle attacks.
```

Comment:
in here you are giving remediation actions which should be in recommendation. The description file should have just the definition of that vulnerability class and the impact of it.

Reply:
this is not remediation it's description of certificate check .

Reply:
I'm not giving remediation actions in there.